Library cooldown policy updates (#3448)
chainguard-dev/edu: content/chainguard/libraries/access.md

 45a0c61a2b3c4d5e6f7g8h9i0j1k2l3m4n5o6p7q/56789abc67890 | PYTHON     | POLICY_CHAINGUARD
```

**Policies** 

To list available policies, run the following:

```bash
chainctl libraries policy list
```

To verify which policy is active, run the following:

```bash
chainctl libraries policy binding list
```

> Prior to `chainctl` version 0.2.291, cooldown policies were enabled via the `chainctl entitlements command`. Cooldown policies from earlier versions are listed as read-only system policies. To change the cooldown period, create a new custom policy and enforce it for a specified ecosystem.

Reviewer: ```suggestion
Prior to `chainctl` version 0.2.291, cooldown policies were enabled via the `chainctl entitlements command`. Cooldown policies configured prior to this version of `chainctl` are migrated under the new `chainctl libraries policies` system. 
```

Reviewer: the old cooldown may be migrated either as a "system" or a "custom" policy. example from my testing:

angelazhang@Mac policy-libs-testing % chainctl libraries policies list
One organization available, selecting angela-staging (id: fea6987fe8946528c1a64508b2ad608c5e811a7f)

       NAME        |  TYPE  | COOLDOWN |                    DESCRIPTION
-------------------|--------|----------|----------------------------------------------------
 migrated-cooldown | CUSTOM | 3        | Migrated cooldown policy: 3-day upstream cooldown.
angelazhang@Mac policy-libs-testing % chainctl libraries policy describe migrated-cooldown
One organization available, selecting angela-staging (id: fea6987fe8946528c1a64508b2ad608c5e811a7f)

ID: fea6987fe8946528c1a64508b2ad608c5e811a7f/1bdbf1358cce7d5e
Name: migrated-cooldown
Type: CUSTOM
Cooldown: 3
Description: Migrated cooldown policy: 3-day upstream cooldown.
Block list (0):
Allow list (0):
